frontend/src/components/punishments/PunishmentGrid.svelte

```diff
@@ -16,7 +16,7 @@
 
   // TODO
   // Remove group_id once members from OW group from backend is implemented.
-  let group_id: number = 1;
+  let group_id: number = 3;
```

Comment:
Selv om det kanskje ikke har så mye å si in the end, så er det greit å holde denne på `1` for nå. Gjør det også vanskelig å teste siden man må lage tre grupper.

Reply:
yeap! agreed!